### DISCUSSION TITLE
PR's not being created, but new branch is

### DISCUSSION THREAD

**Discussion by JClackett** [renovatebot/renovate#8702] [Request Help]
Recently I noticed that pr's are not being created by renovate? A new branch is being created with the correct change but the pr isn't, I didn't change anything with the config since it was working, wondering if somethings changed that I missed?

```json
{
  "extends": ["config:base"],
  "baseBranches": ["develop"],
  "automerge": true,
  "major": { "automerge": false },
  "requiredStatusChecks": null,
  "packageRules": [
    {
      "packagePatterns": ["eslint"],
      "groupName": "eslint"
    },
    {
      "packagePatterns": ["@types/"],
      "groupName": "@types"
    },
    {
      "packageNames": ["aws-sdk"],
      "extends": ["schedule:monthly"]
    }
  ],
  "ignoreDeps": ["yarn"]
}
```

**Comment by rarkins** [comment]
I don't see anything wrong with the config. Please take a look at the debug logs and paste anything here that's of interest. Probably you can just skip ahead to where the branch name is mentioned.

**Reply by JClackett** [reply]
Thanks for the reply. I'm trying to access the renovate dashboard to see the logs, but it just hangs on the Netlify authorised screen?

<img width="1440" alt="Screenshot 2021-02-16 at 12 05 55" src="https://user-images.githubusercontent.com/12549124/108054808-68ecaa80-704f-11eb-8e53-96dbabd5f305.png">

**Reply by rarkins** [reply]
Can you open developer tools and see if there are any logs indicating something failed? I just logged out and in again with Chrome so can't see any problem.

**Reply by JClackett** [reply]
Managed to log in now, just started working. will check out the logs and see what I can find, thanks for the help
